templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -190,5 +190,136 @@
 				</div>
 			{{end}}
 		{{end}}
+
+		{{if .Repository.IsDependenciesEnabled}}
+				<div class="ui divider"></div>
```

Comment:
nit: this is indented two levels

Reply:
How do you mean this?

Reply:
The `div` has two more indents than the `if`, please remove one of these indents (likewise for the rest of the `if...end` block)

Reply:
Ahh, ok. Fixed it.